.nvmrc

```diff
@@ -0,0 +1 @@
+7.10.0
```

Comment:
Good catch, yeah! Going to go with `8.9` based on this: 
> Node v8.9.0 marks the transition of Node v8.x into Long Term Support (LTS), with the codename "Carbon". The v8.x line now moves into "Active LTS", and will remain so until April 2019, at which point it will move into "Maintenance LTS" until December 2019. For more info see the Release Schedule.
https://nodejs.org/en/blog/release/v8.9.0/